lib/era_ja/conversion.rb

```diff
@@ -39,8 +40,10 @@ def to_era(format = "%o%E.%m.%d", era_names: ERA_NAME_DEFAULTS)
           str_time = era_year(year - 1911, :taisho, era_names)
         when self.to_time < ::Time.mktime(1989,1,8)
           str_time = era_year(year - 1925, :showa, era_names)
-        else
+        when self.to_time < ::Time.mktime(2019, 5, 1)
           str_time = era_year(year - 1988, :heisei, era_names)
+        else
+          str_time = era_year(year - 2019, :reiwa, era_names)
```

Comment:
@risacan 
It seems to me that `2018` is correct.
If so, please modify it and I hope some spec will be added.

```ruby
irb(main):006:0> Date.new(2019,5,1).to_era
=> "R00.05.01"
```

Reply:
Thank you ! I will fix this 👍

Reply:
Thank you comment! (@MITSUBOSHI )
You are right!

@risacan 
Can you fix it?
and Can you add some tests?

Reply:
@tomiacannondale 
I fixed the year and add some test. Can you check that? 😄  
Please tell me if you need more tests! 💪

Reply:
oh i didnt notice the conflict! 😱  I'll fix that

Reply:
@tomiacannondale 
I fixed some conflict + Add spec for the first day of reiwa and after.

Reply:
Great 😄 

But tests for Time were not added 😢 

Can you fix it?

Reply:
Okay! 👍  I'll add them!

Reply:
@tomiacannondale 
Added time spec for the first day of reiwa and after ✔️ Can you check these? 🙏